src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -376,9 +377,9 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
             {artwork.shippingInfo}
           </Sans>
         )}
-        {artwork.is_for_sale && artwork.priceIncludesTax && (
+        {artworkEcommerceAvailable && artwork.priceIncludesTaxDisplay && (
```

Comment:
This felt like the right logic (based on `shippingInfo`, etc. above). The logic for `is_acquireable` and `is_offerable` includes a check for "For sale", and I don't think we want this to show for works that are inquireable or at auction.